pkg/config/application.go

```diff
@@ -209,3 +209,18 @@ func getApplicationOrg(config *cb.Config, orgName string) (*cb.ConfigGroup, erro
 	}
 	return org, nil
 }
+
+// AddApplicationOrg adds an organization to an existing config's Application configuration.
```

Comment:
```suggestion
// AddApplicationOrg adds an organization to an existing Application configuration.
```

We really need to move away from possessives in the doc. I realize this is not the only place we’re doing it but I wanted to highlight it.